adserver/forms.py

```diff
@@ -554,6 +556,170 @@ class Meta:
+        )
+        self.fields["start_date"].help_text = _("The target start date for this flight")
+        self.fields["end_date"].help_text = _(
+            "The target end date for this flight (it may go after this date)"
+        )
+        self.fields["advertisements"].queryset = (
+            self.old_flight.advertisements.all()
+            if self.old_flight
+            else Advertisement.objects.none()
+        )
+
+    def create_form_helper(self):
+        helper = FormHelper()
+        helper.attrs = {"id": "flight-request-form"}
+        helper.layout = Layout(
+            Fieldset(
+                "",
+                Field("name"),
+                Div(
+                    Div("start_date", css_class="form-group col-lg-6"),
+                    Div("end_date", css_class="form-group col-lg-6"),
+                    css_class="form-row",
+                ),
+                Div(
+                    PrependedText(
+                        "budget",
+                        "$",
+                        min=0,
+                        data_bind="textInput: budget",
+                    ),
+                ),
+                Field("note"),
+                css_class="my-3",
+            ),
+            Fieldset(
+                _("Advertisements"),
+                Field(
+                    "advertisements",
+                    template="adserver/includes/widgets/advertisement-form-option.html",
+                ),
+                css_class="my-3",
+            ),
+            Submit("submit", _("Request a new flight")),
+            HTML(
+                "<p class='form-text small text-muted'>"
+                + str(
+                    _(
+                        "Your flight will not start automatically. Your account manager will be notified to review your ads and targeting."
+                    )
+                )
+                + "</p>"
+            ),
+        )
+        return helper
+
+    def save(self, commit=True):
+        assert commit, "Delayed saving is not supported on this form"
+
+        # This is already the default but we are setting it explicitly to be defensive
+        self.instance.live = False
+
+        if self.old_flight:
+            # Copy fields not in the form from the old flight if possible
+            # Otherwise, the account manager will need to set these
+            fields = (
+                "targeting_parameters",
+                "priority_multiplier",
+                "cpm",
+                "cpc",
+                "sold_impressions",
+                "sold_clicks",
+            )
+            for field in fields:
+                setattr(self.instance, field, getattr(self.old_flight, field))
+
+        # We must set the campaign
+        self.instance.campaign = (
+            self.old_flight.campaign
+            if self.old_flight
+            else self.advertiser.campaigns.all().first()
```

Comment:
```suggestion
            else self.advertiser.campaigns.first()
```

Don't think the `all()` is needed.